fleetspeak/src/server/components/prometheus/prometheus.go

```diff
@@ -186,15 +187,30 @@ var (
 	)
 )
 
+// Returns a well-defined string representing labels from a given client.
```

Comment:
Should this "well-defined string" be its own type instead?

(can be done in a separate PR)

Reply:
It's more that it's an unambiguous and stable representation of the labels. In this particular case I'm not sure how much creating its own type would bring us, since we're talking about a private utility function that's essentially an implementation detail.